fluffy/static/js/home.js

```diff
@@ -27,12 +27,14 @@ $(document).ready(function() {
             fh.slideUp(duration);
             pb.slideDown(duration, function() {
                 transitioningModes = false;
+                $(document).off("paste", pastefile);
```

Comment:
maybe just add the equivalent of `if (!fh.is(':visible')) return;` to the handler and don't worry about turning it on/off all the time?